src/console/MeasurementCollectionToFhir/Processor.cs

```diff
@@ -0,0 +1,60 @@
+﻿// -------------------------------------------------------------------------------------------------
```

Comment:
instead of writing these Processor/ProcessorStartup classes, it may be easier to take the existing Functions code in IomtConnectorFunctions and then just remove the EventHub triggers for our use case here. That way we can reuse as much code as possible and would probably not need these separate ProcessorStartup DI classes.

The reason I didn't do this originally is that I didn't want to take a dependency on Webjobs. But later I found that keeping webjobs would allow us to keep the blob triggers which are pretty useful and also allow us to reuse the existing oss code in a more straightforward way, so I will be adding the webjobs dependency back and adding the blob triggers back (and will remove the hard coded mapping templates)

Reply:
I ended up not doing this due to time, and not sure it is worth while given that the blob inputs aren't triggers. Ended up just fetching the blob items from storage each event batch to replicate current functions behavior.